corehq/apps/hqcase/management/commands/ptop_fast_reindexer.py

```diff
@@ -159,13 +166,35 @@ def load_from_view(self):
             fout.write(str(current_db_seq))
 
         # Load entire view to disk
+        if self.sort_key:
+            self._load_from_view_and_sort()
+        else:
+            self._load_from_view()
+        self.log("View and sequence written to disk: %s" % datetime.utcnow().isoformat())
+
+    def _load_from_view_and_sort(self):
```

Comment:
took a little while to parse through this. maybe add a doc string with something like: "writes a tab delimited file where the first key is the sort key and the second is the row (either doc or ID), then sorts by the sort key and removes it from the final output file leaving just the row" (assuming i read it right)